### ISSUE TITLE
Functions customer would like to use Microsoft.Xrm.Data.PowerShell in PowerShell Core 6

### ISSUE THREAD

**Issue by Francisco-Gamino** [seanmcne/Microsoft.Xrm.Data.PowerShell#372]
Currently, In Azure Functions we support PowerShell 6. However, Microsoft.Xrm.Data.Powershell only works on Full CLR PowerShell (Version 5). Are there any plans to port this module to CoreCLR?

In theory, this module might work in PowerShell 7, which is built on top of .NetCore 3 (https://docs.microsoft.com/en-us/dotnet/core/whats-new/dotnet-core-3-0). However, PowerShell 7 is in preview and it is going to be a while before it GAs.

Repro steps for PowerShell Core 6:

* Install PowerShell Core 6
* Download Microsoft.Xrm.Data.Powershell module from https://www.powershellgallery.com/packages/Microsoft.Xrm.Data.Powershell/2.8.0

```
PS C:\Modules> mkdir Modules
PS C:\Modules> cd .\Modules\
PS C:\Modules> Save-Module -Name Microsoft.Xrm.Data.Powershell -Path . -Verbose
PS C:\Modules> Import-Module .\Microsoft.Xrm.Data.Powershell\
Import-Module : Could not load type 'System.Windows.DependencyObject' from assembly 'WindowsBase, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'.
At line:1 char:1
+ Import-Module .\Microsoft.Xrm.Data.Powershell\
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : NotSpecified: (:) [Import-Module], TypeLoadException
+ FullyQualifiedErrorId : System.TypeLoadException,Microsoft.PowerShell.Commands.ImportModuleCommand
```

**Comment by seanmcne** [comment]
We are blocked by CrmServiceClient, unfortunately. When the PG updates that and the xrm sdk to support .NET core, we can look at doing this.

**Comment by seanmcne** [comment]
Closing in favor of powershell 7 duplicate request (same scenario, .NET core support currently blocked by the product connector).
